librubyfmt/Cargo.toml

```diff
@@ -12,6 +12,7 @@ serde = { version = "1.0", features=["derive"] }
 serde_json = "1.0.40"
 backtrace = "0.3.45"
 libc = "0.2.68"
+fancy-regex = "0.14.0"
```

Comment:
N.B. the `regex` crate doesn't support lookbacks, which we originally used in the `rubyfmt_lib` implementation, hence this uses `fancy-regex` instead (which is really just the `regex` crate plus lookbacks and other regex features that operate in greater-than-linear time).